olive/systems/azureml/aml_pass_runner.py

```diff
@@ -89,6 +89,11 @@ def main(raw_args=None):
 
     # save model json
     model_json = output_model.to_json()
+
+    # Replace output model HF config with input model HF config
```

Comment:
Since, we are planning to make the passes autoinsert the hf_config, are we still keeping this section?

Reply:
Or we remove this when that PR happens?

Reply:
This is needed for aml run for now. We should remove this after we make the changes on passes